### PR TITLE
announce self-hosted hive

### PR DESCRIPTION
**Pull request by n1ru4l**

### CODE REVIEW

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+title: Announcing self-hosted GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: [laurin]
+date: 2022-10-04
+description: Host your own GraphQL Hive instance.
+image: /blog-assets/announcing-self-hosted-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-self-hosted-graphql-hive/thumbnail.png
+---
+
+It has been 5 months since [we first announced GraphQL Hive](/blog/graphql-hive-preview) and it has been a wild ride since then!
```
Comment (kamilkisiela) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:11]: `Mar 26th 2021`

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+> GraphQL Hive is our schema registry solution for continously evolving your GraphQL schema backed by performance insights and usage statistics.
+> Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
```
Comment (kamilkisiela) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:14]: @B2o5T Don't we have quotes? Is it missing or we never had them?

Zrzut ekranu 2022-10-4 o 14 37 50

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+We frist were surprised but also excited about the amount of interest from companies of all sizes and industries and individuals from all over the world in GraphQL Hive.
+We are so grateful for all the support, contributions and feedback we received so far from external contributors and are looking forward to a bright future!
+
+Aside from the diverse range of companies that are already successfully using our hosted GraphQL Hive version, a lot of companies have been asking us whether it is possible to host your own GraphQL Hive instance!
+
+Since the beginning of the project, we had a different vision for SaaS and community editions of GraphQL Hive compared to traditional SaaS products.
+Instead of only open-sourcing a small portion of the whole stack we always wanted to open-source the full stack and make it easy for anyone to host their own instance of GraphQL Hive.
+
+Today, we are happy to announce that we have reached this milestone and spinning up your first GraphQL Hive instance is as easy as running a single command!
+All you need is `docker` and `docker-compose` installed on your machine.
```
Comment (kamilkisiela) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:25]: isn't `$ docker-compose` now `$ docker compose` (built-in)?

**Review by kamilkisiela — APPROVED** [review]
I added some comments but it looks goooooood

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+image: /blog-assets/announcing-self-hosted-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-self-hosted-graphql-hive/thumbnail.png
+---
+
+It has been 5 months since [we first announced GraphQL Hive](/blog/graphql-hive-preview) and it has been a wild ride since then!
+
+> GraphQL Hive is our schema registry solution for continously evolving your GraphQL schema backed by performance insights and usage statistics.
+> Whether you are working in a small company with a single GraphQL service or a big multi-team company with multiple (federated or composed) GraphQL services, GraphQL Hive is the perfect tool to help you manage your GraphQL schema.
+
+We frist were surprised but also excited about the amount of interest from companies of all sizes and industries and individuals from all over the world in GraphQL Hive.
```
Comment (charlypoly) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:16]:
```suggestion
We first were surprised but also excited about the amount of interest from companies of all sizes and industries and individuals from all over the world in GraphQL Hive.
```

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx (hunk 1 above)
Comment (kamilkisiela) [reply website/pages/blog/announcing-self-hosted-graphql-hive.mdx:11]: Maybe we should link to https://4e7b143f.guild-dev-website.pages.dev/blog/announcing-graphql-hive-release? It's when we went public and mention the preview program (/blog/graphql-hive-preview)?

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx (hunk 2 above)
Comment (B2o5T) [reply website/pages/blog/announcing-self-hosted-graphql-hive.mdx:14]: @kamilkisiela try to clear cache, it's fine for me 
image

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+Aside from the diverse range of companies that are already successfully using our hosted GraphQL Hive version, a lot of companies have been asking us whether it is possible to host your own GraphQL Hive instance!
+
+Since the beginning of the project, we had a different vision for SaaS and community editions of GraphQL Hive compared to traditional SaaS products.
+Instead of only open-sourcing a small portion of the whole stack we always wanted to open-source the full stack and make it easy for anyone to host their own instance of GraphQL Hive.
+
+Today, we are happy to announce that we have reached this milestone and spinning up your first GraphQL Hive instance is as easy as running a single command!
+All you need is `docker` and `docker-compose` installed on your machine.
+
+[Head over to our GraphQL Hive documentation for getting started!](https://docs.graphql-hive.com/self-hosting/get-started)
```
Comment (B2o5T) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:27]: link is broken
image

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx (hunk 3 above)
Comment (n1ru4l) [reply website/pages/blog/announcing-self-hosted-graphql-hive.mdx:25]: didn't use `docker compose` yet :D

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx (hunk 5 above)
Comment (n1ru4l) [reply website/pages/blog/announcing-self-hosted-graphql-hive.mdx:27]: @kamilkisiela Seems like the docs did not get redeployed 👀  https://docs.graphql-hive.com/

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing self-hosted GraphQL Hive
+tags: [graphql, graphql-hive]
+authors: [laurin]
+date: 2022-10-04
```
Comment (dotansimha) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:5]: update before merging

**Review by dotansimha — APPROVED** [review]

File: website/pages/blog/announcing-self-hosted-graphql-hive.mdx (hunk 6 above)
Comment (n1ru4l) [inline website/pages/blog/announcing-self-hosted-graphql-hive.mdx:5]:
```suggestion
date: 2022-10-05
```

**Pull request merged**